### ISSUE TITLE
Git Tags

### ISSUE THREAD

**Issue by franz-josef-kaiser** [openatv/enigma2#1636]
While I was in the process of documenting my setup, I wanted to set recurring personal calendar events to see if there's something to update. Browsing through the list of [releases](https://github.com/openatv/enigma2/releases), I can see that this concept seems to have been abandoned. The list of _git tags_ ends with [5.3](https://github.com/openatv/enigma2/tags) in 2017. There seems to be some branch model that is hard to follow (I have to clone the repo and look at the latest commits locally without a sure release date).

*Questions:*
1. Is there a changelog?
1. Can I help tagging this repo? (Where can I start? Latest commit on branch x.x? ~~I am not sure if I can send a PR for tags only.~~ If you trust me enough to give me temp contributor access, I would be willing to search and tag previous versions.)
1. Does this repo follow the [semver](https://semver.org/) standard? _For e.g.: 1.1.1 == Major.Minor.Patch version._

Thanks in advance!

**Comment by atvcaptain** [comment]
branch 6.4 are the current version

you can send pull requst to this branch

**Comment by franz-josef-kaiser** [comment]
Hi @atvcaptain ,
my question was not about where I can send pull requests, but about missing Git Tags. I offered to add them if no one is willing to add them. Can do this with temporary/ short term contributor access to this repo.

**Comment by franz-josef-kaiser** [comment]
@atvcaptain pinging you. Long weekend ahead and I could invest some time.

**Comment by arn354** [comment]
A tag does not say much  - at least how openatv is build.
There is once a official first release of for example 6.4 - we have no incremental buildversions. 6.4 is 6.4 and is build nightly from repo branch 6.4 latest commit. So every night there is a new release - depending on build speed/time for buildcycle trough all boxes.
Not sure how further tags would improve something.

**Comment by arn354** [comment]
if you want to build 6.3 - you need to clone oe-alliance 4.3
If you want to build 6.4 - you need to clone oe-alliance 4.4
and you always build the latest commit of openatv enigma2 in the corresponding branch of oe-alliance build-environment.

**Comment by franz-josef-kaiser** [comment]
@arn354 Thanks for clearing things up a bit. As some things are still a bit confusing and I seem to be not fully able to wrap my head around this onion approach with `oe-alliance` and `openatv` orgs here on GitHub, would you mind explaining the stack a bit more in detail? I could not find any docs, but just a link that explains the setup would be fine too. Thanks in advance!

**Comment by arn354** [comment]
https://github.com/openatv/enigma2/blob/6.4/README.md - this explains what you have to do if you want to build openatv for a box.

required build-environment with according sub-modules is there https://github.com/oe-alliance/build-enviroment. 
brand, distro, etc. stuff is located in oe-alliance-core - https://github.com/oe-alliance/oe-alliance-core (a submodule of build-environment)
there are more repositories a complete image is build from.

**Comment by jbleyel** [comment]
We don’t use git tags because we do not have minor versions. Each major version is a new branch.
